project/templates/pronos.html

```diff
@@ -40,11 +40,39 @@ <h1 class="title has-text-white is-capitalized"></h1>
                 <h1 class="title has-text-white">Matchs du {{leaguename}}</h1>
             </div>
         </div>
-        {% for item in records %}
-        <div class="box has-background-dark has-text-white">
-            <h1 class="title is-4 has-text-white ">{{item}}</h1>
-        </div>
-        {% endfor %}
+        <form method="POST" action="/pronos_update">
+            <button class="button is-link is-dark is-medium">Valider mes pronostics</button>
+                <h1 class="title has-text-white">                </h1>
+            {% for item in records %}
+            <div class="box has-background-dark has-text-white">
+                <div class="columns">
+                    <div class="column">{{item.game_datetime}} (BO{{item.bo}})</div>
+                    <div class="column">{{item.team_1}}</div>
+                    <div class="column">{{item.score_team_1}}</div>
+                    <div class="column">{{item.score_team_2}}</div>
+                    <div class="column">{{item.team_2}}</div>
+                    <div class="column">Points</div>
+                </div>
+                <div class="columns has-background-grey-lighter has-text-dark">
+                    <div class="column"></div>
+                    <div class="column has-text-large">Mon pronostic : </div>
```

Comment:
On teste si le pronostic est modifiable, si oui on ne change rien.
```suggestion
                    <div class="column has-text-large">Mon pronostic : </div>
                    {% if item.editable %}
```